File: 0326-power-of-three/0326-power-of-three.cpp
class Solution {
public: 
    bool recur(int n){
        if(n==1)return 1;
        if(n%3!=0 or n==0)return 0;
        return recur(n/3);
    }
    bool isPowerOfThree(int n) {
        return recur(n);
    }
};

File: 0326-power-of-three/0326-power-of-three.java
class Solution {
    public boolean isPowerOfThree(int n) {
        if(n<=0)return false;
        double d=Math.ceil(Math.log(n)/Math.log(3));
        System.out.println(d);
        if(Math.pow(3,(int)d)==(double)n)return true;
        return false;
    }
}